packages/api/src/client/internal/generated/openapi/namespaces/OntologiesV2.ts

```diff
@@ -0,0 +1,971 @@
+import type { AttachmentV2 } from "../components/AttachmentV2";
+import type { TimeSeriesPoint } from "../components/TimeSeriesPoint";
+import type { StreamTimeSeriesPointsRequest } from "../components/StreamTimeSeriesPointsRequest";
+import type { ApplyActionRequestV2 } from "../components/ApplyActionRequestV2";
+import type { SyncApplyActionResponseV2 } from "../components/SyncApplyActionResponseV2";
+import type { AsyncApplyActionRequestV2 } from "../components/AsyncApplyActionRequestV2";
+import type { AsyncApplyActionResponseV2 } from "../components/AsyncApplyActionResponseV2";
+import type { PreviewMode } from "../components/PreviewMode";
+import type { ListQueryTypesResponseV2 } from "../components/ListQueryTypesResponseV2";
+import type { QueryApiName } from "../components/QueryApiName";
+import type { QueryTypeV2 } from "../components/QueryTypeV2";
+import type { ExecuteQueryRequest } from "../components/ExecuteQueryRequest";
+import type { ExecuteQueryResponse } from "../components/ExecuteQueryResponse";
+import type { ObjectSetRid } from "../components/ObjectSetRid";
+import type { ObjectSet } from "../components/ObjectSet";
+import type { LoadObjectSetRequestV2 } from "../components/LoadObjectSetRequestV2";
+import type { LoadObjectSetResponseV2 } from "../components/LoadObjectSetResponseV2";
+import type { AggregateObjectSetRequestV2 } from "../components/AggregateObjectSetRequestV2";
+import type { AggregateObjectSetResponseV2 } from "../components/AggregateObjectSetResponseV2";
+import type { ListDeploymentsResponse } from "../components/ListDeploymentsResponse";
+import type { DeploymentApiName } from "../components/DeploymentApiName";
+import type { DeploymentMetadata } from "../components/DeploymentMetadata";
+import type { TransformDataRequest } from "../components/TransformDataRequest";
+import type { TransformDataResponse } from "../components/TransformDataResponse";
+import { OpenApiRequest } from "../request";
+
+/**
+ * Lists the Ontologies visible to the current user.
+ *
+ * Third-party applications using this endpoint via OAuth2 must request the following operation scope: `api:read-data`.
+ *
+ */
+export function listOntologiesV2<TResponse>(_request: OpenApiRequest<ListOntologiesV2Response, TResponse>): Promise<TResponse> {
+    return _request(
+        "listOntologiesV2",
+        "GET",
+        "/v2/ontologies",
+        __undefined,
+        __undefined,
+        __undefined,
+        __undefined,
+        __undefined,
+        "application/json",
+    );
+}
+
+/**
+ * Gets a specific ontology with the given Ontology RID.
+ *
+ * Third-party applications using this endpoint via OAuth2 must request the following operation scope: `api:read-data`.
+ *
+ */
+export function getOntologyV2<TResponse>(_request: OpenApiRequest<OntologyV2, TResponse>, ontology: OntologyApiName): Promise<TResponse> {
+    return _request(
+        "getOntologyV2",
+        "GET",
+        "/v2/ontologies/{ontology}",
```

Comment:
Given we have template literals, does it make sense to have a special section for passing along query params vs just putting them in place?